components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/dao/OAuthAppDAO.java

```diff
@@ -1309,9 +1309,9 @@ private boolean isOIDCAudienceEnabled() {
     }
 
     private void handleRequestForANonExistingConsumerKey(String consumerKey) throws InvalidOAuthClientException {
-        String message = "Cannot find an application associated with the given consumer key : " + consumerKey;
+        String message = "application.not.found";
         if (LOG.isDebugEnabled()) {
-            LOG.debug(message);
+            LOG.debug("Cannot find an application associated with the given consumer key.");
```

Comment:
Is there any reason to not to add `consumerKey` into the debug log.

Reply:
Fixed by https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1850.